packages/core/aqua-compiler/src/generate/__test__/snapshots/smoke_test.js

```diff
@@ -0,0 +1,763 @@
+/* eslint-disable */
+// @ts-nocheck
+/**
+ *
+ * This file is auto-generated. Do not edit manually: changes may be erased.
+ * Generated by Aqua compiler: https://github.com/fluencelabs/aqua/.
+ * If you find any bugs, please write an issue on GitHub: https://github.com/fluencelabs/aqua/issues
+ * Aqua version: 0.12.0
```

Comment:
if this snapshot is not updated. Why your tests still pass?

Reply:
Probably error in tests, should be fixed for now